compiler/luci-interpreter/src/loader/KernelBuilder.cpp

```diff
@@ -74,8 +75,13 @@
 #include "kernels/Transpose.h"
 #include "kernels/TransposeConv.h"
 
+#include "Halide.h"
```

Comment:
luci-interpreter is used for quantization tool and this is included in the release package.
if we include also halide, we need to include halide package to our release package,
which I don't know how this is simple or complicated, and there may be concern with version control.
could you please check how this is turn out?

or if you could make this optional so that our tools can be released without halide then that is also OK.

Reply:
Yes, I thought about this, and I think I can remove dependency, or at least leave minimal set of struct declarations (there are not many of them, afaik).
I included whole halide for now just because this was faster and easier and I want to get working prototype as soon as possible. =)

If I fail to remove dependency I have two things on my mind:
1) Easy and fast way: make compiled node support optional.
2) Harder and slower way: try to remove/wrap halide related stuff on compiler-side.

P.s. Actually I would like to proceed with 2 case, because this eliminates redundant dependencies entirely.